portal/templates/profile_base.html

```diff
@@ -0,0 +1,61 @@
+{%- extends "layout.html" -%}
+{%- from "flask_user/_macros.html" import linksHTML, logo -%}
+{%- block main -%}
+<form id="profileForm" class="form tnth-form to-validate" data-toggle="validator">
+  <input type="hidden" id="profileUserId" value="{{user.id if user}}" />
+  <input type="hidden" id="profileCurrentUserId" value="{{current_user.id if current_user}}" />
+  <div class="row">
+    <div class="col-lg-12 col-md-12 col-sm-12">
+      <br/>
+      <div class="right-panel">
+        <div class="flex">
+          <div id="profileHeader" class="profile-header">
+            <h4 class="tnth-headline">
+              {% block profile_title %}
+              {{_("TrueNTH Profile")}}
+              {% endblock %}
+            </h4>
+          </div>
+        </div>
+        {% block profile_content %}
+        {% endblock %}
+        <br/><br/>
+      </div>
+    </div>
+  </div>
+</form>
+{% endblock %}
+{% block footer %}<div class="footer-wrapper right-panel"><div class="flex footer-container"><div class="copyright-container">{%-if providerPerspective== 'true' -%}{{linksHTML(user=current_user)}}{%-else-%}{{linksHTML(user=user)}}{%-endif-%}</div><div class="logo-container">{{logo(True)}}</div></div></div>{% endblock %}
+{% block additional_scripts %}
+<script src="{{ url_for('static', filename='js/procedures.js') }}" async></script>
+{% endblock %}
+{% block document_ready %}
+  $("#mainDiv").addClass("profile");
```

Comment:
Can this be moved into a separate file (I think it appears in an inline script)?